### PR TITLE
Ezaf 4067 | EZAF-3639 | RAY GPU EXAMPLE | RAY 2.7.0

### PR DESCRIPTION
**Pull request by sip-aravind-g**
This example show how the RAY GPU works. using the RAY submission approach.

### CODE REVIEW

**Review by sercanCyberVision — APPROVED** [review]
Thank you @sip-aravind-g, LGTM!

File: Data-Science/Ray-CPU/run_cpu.ipynb [hunk 1]
```diff
@@ -75,7 +75,13 @@
     "get_job_logs = client.get_job_logs(job_id)\n",
     "get_job_info = client.get_job_info(job_id)\n",
     "async for lines in client.tail_job_logs(job_id):\n",
+<<<<<<< HEAD
```
Comment (ask664) [inline Data-Science/Ray-CPU/run_cpu.ipynb:78]: Please resolve conflict.

Comment (Manish-2004) [reply Data-Science/Ray-CPU/run_cpu.ipynb:78]: Done

Comment (sip-aravind-g) [reply Data-Science/Ray-CPU/run_cpu.ipynb:78]: its done @ask664

**Comment by ask664** [comment]
Did we test with latest build?

**Comment by ask664** [comment]
even we need to rebuild ezua-tutorial image and we need to raise PR on ezaf and hefo repository as well

**Comment by sip-aravind-g** [comment]
We have informed QA folks and conveyed our PR link for testing, Until the examples are consolidated into the EzAF build.

**Comment by ask664** [comment]
We can use QA env for testing purpose because we will rebuild image again and again, please connect with QA team and test it.

**Review by Manish-2023 — APPROVED** [review]

**Pull request merged**